src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -645,9 +645,10 @@ private static void getFormat(StreamType streamType, DLNAMediaInfo media, DLNAMe
 			format = FormatConfiguration.BMP;
 		} else if (value.equals("tiff")) {
 			format = FormatConfiguration.TIFF;
+		// Value can look like "Advanced@L1", "Complex@L2", "MP@LL" or "Simple@L3" with VC-1 or MPEG-4 Visual at this point.
 		} else if (containsIgnoreCase(value, "@l") && streamType == StreamType.Video) {
-			media.setAvcLevel(getAvcLevel(value));
-			media.setH264Profile(getAvcProfile(value));
+		//TODO: Is this really needed, it's already set earlier..?
```

Comment:
@Sami32 That was the core of my change, instead of storing H264Level, H264Profile, H265Level and H265Profile (having to add yet more fields for other standards if needed) I simply store the unparsed string as returned from MediaInfo. Then, when one of the values above is needed, they are parsed from this string. I called it "format profile" because that's the name MediaInfo use, and I really didn't know a more suitable name. It's more general that when you think of, which is the "profile" part of H264 or H265.